docs/django-widget.rst

```diff
@@ -59,6 +67,51 @@ widget url:
         'upload_base_url' = 'http://path.to/your/upload/handler',
     }
 
+Use ``widget_options`` to pass arbitrary `options`_ to the file uploader:
+
+.. code-block:: python
+
+    UPLOADCARE = {
+        # ...
+        'widget_options' = {
+            'camera-mirror': True,
+            'source-list': 'local, camera',
```

Comment:
Let's add "from URL" by default too, and put source-list on top (for a better readability).